src/Blocks/BlockSnow.h

```diff
@@ -88,6 +88,16 @@ class cBlockSnowHandler :
 		UNUSED(a_Meta);
 		return 14;
 	}
+
+	virtual double GetBlockHeight(void) override
+	{
+		return 1.0 / 8.0;  // one layer of snow
```

Comment:
The function might need a `a_BlockMeta` param in order to be able to handle snow properly - multiple layers of snow have a different height.